### ISSUE TITLE
Has the comparison algorithm been retrained?

### ISSUE THREAD

**Issue by Unitshockresponse** [PeterL1n/BackgroundMattingV2#101]
You compared DIM and FBA algorithms. Have you retrained them on the VideoMatte240K, PhotoMatte13K, and Distinctions-646 data sets. Or directly use the pre-trained model provided by the original author to test.

**Comment by PeterL1n** [comment]
No. Only BackgroundMattingV1 were retrained for comparison

**Comment by Unitshockresponse** [comment]
> No. Only BackgroundMattingV1 were retrained for comparison
Thank you for your reply. If the training sets are different, might it be unfair? What do you think

**Comment by PeterL1n** [comment]
No didn't have time to train every methods. Additionally, FBA uses trimap, we already acknowledged with get trimap, then method does better. It was a different category of method and not the focus of our study.

**Comment by Unitshockresponse** [comment]
I have an idea, if DIM is jointly trained on three data sets. Can its indicators be better? DIM may also exceed your indicators. In other words, adding a background image can't exceed the existing trimap-based algorithm?
Anyway, BGV2 is still a great job.
